README.md

```diff
@@ -98,5 +98,8 @@ apl
 * sof-kernel-dump.sh
 <br> catch all kernel information after system boot up from /var/log/kern.log file
 
+* sof-get-kernel-line.sh
+<br> catch all kernel line number from /var/log/kern.log file
```

Comment:
After run a test I got what you mean here, I shows the start line number with the kernel version form oldest to latest. I will suggest to from latest to oldest.
@stripes416 Could you help to get some more accurate description for this script.
```
22 4.15.0-74-generic
1092 4.15.0-74-generic
2156 4.15.0-74-generic
3246 4.15.0-74-generic
```

Reply:
```suggestion
<br> print all kernel versions and their line numbers from /var/log/kern.log, with the most recent <first/last>
```
first/last is just based on whichever way you choose to print it.